lib/kitto/notifier.ex

```diff
@@ -44,6 +44,7 @@ defmodule Kitto.Notifier do
   Emits a server-sent event to each of the active connections with the given
   topic and payload
   """
+  def broadcast!(data, topic) when is_atom(topic), do: broadcast!(topic, data)
```

Comment:
I was going to ask you to add a test for this, but we don't have a test file for this module.